browser/ui/color/brave_color_mixer.cc

```diff
@@ -477,9 +474,7 @@ void AddBraveDarkThemeColorMixer(ui::ColorProvider* provider,
   mixer[kColorFeaturePromoBubbleCloseButtonInkDrop] = {
       GetToolbarInkDropColor(mixer)};
 
-  if (base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs)) {
     tabs::AddBraveVerticalTabDarkThemeColorMixer(provider, key);
```

Comment:
clang-format

Reply:
Oops, thanks 😄

Hum, It could be just me but npm run format doesn't work. Um, if npm run format works, shouldn't CI catch this from lint stage?

Reply:
> Oops, thanks 😄
> 
> Hum, It could be just me but npm run format doesn't work. Um, if npm run format works, shouldn't CI catch this from lint stage?

I think in this exact case it didn't work, because it looks only at the changed lines. It looks like a tiny edge case that's not covered by the upstream format implementation.